libs/tokens/src/state/environmentAtom.ts

```diff
@@ -4,6 +4,7 @@ import { atomWithPartialUpdate, getCurrentChainIdFromUrl } from '@cowprotocol/co
 
 interface TokensModuleEnvironment {
   chainId: SupportedChainId
+  useUniswapListOnly?: boolean
```

Comment:
nit: maybe we could use a more generic name that doesn't expose the technical implementation. `useCuratedList`?

Reply:
Yes, it matches better.
Thanks! Renamed